src/options.ts

```diff
@@ -32,6 +32,7 @@ import { nitroImports } from "./imports";
 const NitroDefaults: NitroConfig = {
   // General
   debug: isDebug,
+  timing: isDebug,
```

Comment:
We might as well have it in `nitro-dev` preset (reason is that i am going to support non node dev experimental presets and timing feature is not that stable but can do later...)

Reply:
Interesting ! Non node dev presets support !? 😏